src/components/clusterDetail/ClusterDetail.tsx

```diff
@@ -120,6 +122,13 @@ const ClusterDetail: React.FC<ClusterDetailProps> = ({ cluster }) => {
         >
           Close
         </ToolbarButton>
+        {isLaunchModalOpen && (
```

Comment:
It is better to use Modal isOpen prop instead of this

Reply:
done